### PR TITLE
feat(Transfer): change dialog to new design

### PR DESCRIPTION
**Pull request by lucbpz**
## What does this PR do?

- Updates design of Transfer dialog to the new one.

image

### CODE REVIEW

**Review by felipecruz91 — COMMENTED** [review]
Can the dropdown options be listed downwards instead?

image

File: ui/src/components/TransferDialog.tsx [hunk 1]
```diff
@@ -139,43 +138,43 @@ export default function TransferDialog({ ...props }) {
...
+              To:
+            </Typography>
+          </FormLabel>
+          <Stack mt={1} spacing={3}>
+            <TextField
+              fullWidth
+              autoFocus
+              id="dest-host"
+              label="Destination host"
+              helperText="SSH must be enabled and configured between the source and destination Docker hosts"
```
Comment (felipecruz91) [inline ui/src/components/TransferDialog.tsx:167]: nit: shall we be more explicit? e.g:

"If you're having trouble retrieving the list of volumes from your remote Docker host, check that you've added the ssh public key in the `known_hosts` file."

**Comment by lucbpz** [comment]
> nit: Can the dropdown options be listed downwards instead?

@felipecruz91 I would not change the way the Autocomplete handles the positioning, as it tries to handle that in a smart way, depending on the height of the parent, and also if scrollable, handles that automatically depending on the position. In my case for example, it is displayed downwards.

image

File: ui/src/components/TransferDialog.tsx (hunk 1 above)
Comment (lucbpz) [reply ui/src/components/TransferDialog.tsx]: I kept the first sentence and added the second part of your suggestion, wdyt?

`SSH must be enabled and configured between the source and destination Docker hosts. Check you have the remote host SSH public key in your known_hosts file.`

Comment (felipecruz91) [reply ui/src/components/TransferDialog.tsx]: Perfect!

**Pull request merged**